### PR TITLE
Add CSS progress for right column

### PR DESCRIPTION
**Pull request by valevassallo**

### CODE REVIEW

**Comment by MariethPerez** [comment]
Good Job!!!

File: public/index.html [hunk 1]
```diff
@@ -1,50 +1,90 @@
...
+        <div class="bell">
+          <i class="far fa-bell"></i>
+        </div>
       </div>
-      <div class="bell">
-        <i class="far fa-bell"></i>
+      <div class="jump-to">
+        <div class="left-part">
+          <img
+            src="https://www.iconsdb.com/icons/preview/white/search-xxl.png"
```
Comment (stg101) [inline public/index.html]: Why are you using  an external url for this?

Comment (valevassallo) [reply public/index.html]: Because I couldn't find a better icon

File: public/style.css [hunk 2]
```diff
@@ -196,4 +196,80 @@ span {
   flex-direction: column;
   height: 100vh;
   justify-content: space-between;
+  align-items: center;
```
Comment (AngieCristina) [inline public/style.css]: Your work is very similar to the original Slack, I think you should continue making a great job, how have you been doing.

**Pull request merged**